static/js/presence.js

```diff
@@ -40,7 +41,12 @@ export function is_active(user_id) {
 
 export function get_status(user_id) {
     if (people.is_my_user_id(user_id)) {
-        return "active";
+        if (user_settings.presence_enabled === undefined || user_settings.presence_enabled) {
```

Comment:
What's the story with `user_settings.presence_enabled === undefined`?  I'm not aware of a reason that setting would ever have that value.  Its type appears to be just a straightforward boolean.

Reply:
You're seem to be right. I couldn't reproduce any case that leads to an `undefined` with the actual code on the dev env, so I removed that bit and also changed the node_tests to setup that value before testing.